include/circt/Dialect/CHALK/CHALKOps.td

```diff
@@ -0,0 +1,65 @@
+//===- CHALKOps.td - Definition of CHALK dialect operations -------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_DIALECT_CHALK_CHALKOPS_TD
+#define CIRCT_DIALECT_CHALK_CHALKOPS_TD
+
+include "mlir/IR/OpAsmInterface.td"
+include "mlir/IR/BuiltinAttributes.td"
+include "mlir/Interfaces/ControlFlowInterfaces.td"
+
+def CellOp : CHALKOp<"cell", [
+    SingleBlock,
+    IsolatedFromAbove,
+    NoTerminator
+  ]> {
+  let summary = "Define a 2D VLSI layout";
+  let description = [{
+    `chalk.cell` represents a 2D VLSI layout, where the region nests
+    multiple shapes, paths, or placements.
+  }];
+
+  let arguments = (ins StrAttr:$cell_name);
+  let results = (outs);
+  let regions = (region VariadicRegion<SizedRegion<1>>:$body);
```

Comment:
Is the intent to support multiple regions here? I don't think I've seen any uses of VariadicRegion.

Reply:
No, my original intent was to make the region entirely optional, but that may not be appropriate in this case.  I've dropped the constraint and updated the filecheck test.